### PR TITLE
Remove jobs failing for more than one week

### PR DESCRIPTION
**Pull request by halamix2**
**Description**

Changes proposed in this pull request:

- Remove jobs described in #7462, which have 100% failure rate for the last week or longer

**Related issue(s)**

### CODE REVIEW

**Comment by halamix2** [comment]
I've talked with the Busola team, `post-busola-smoke-test-docker` can be removed

File: prow/jobs/kyma/kyma-integration.yaml [hunk 1]
```diff
@@ -1167,60 +1110,6 @@ periodics: # runs on schedule
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
-    - name: serverless-function-metrics-generator
```
Comment (kwiatekus) [inline prow/jobs/kyma/kyma-integration.yaml:1170]: Please keep it. 
@kyma-project/otters will fix it
https://github.com/kyma-project/test-infra/issues/7684

**Review by kwiatekus — CHANGES_REQUESTED** [review]

File: prow/jobs/kyma/kyma-integration.yaml (hunk 1 above)
Comment (halamix2) [reply prow/jobs/kyma/kyma-integration.yaml]: Restored

**Review by Disper — APPROVED** [review]
@kyma-project/framefrog team does not rely on any of those jobs.

**Comment by halamix2** [comment]
Rebased with main, as one CLI job and Busola smoke tests were removed in #7669

**Comment by halamix2** [comment]
/hold
Until Thursday

File: prow/jobs/kyma/kyma-integration.yaml (hunk 1 above)
Comment (dbadura) [reply prow/jobs/kyma/kyma-integration.yaml]: Fixed: https://github.com/kyma-project/test-infra/pull/7685

**Review by kwiatekus — APPROVED** [review]

**Review by dekiel — APPROVED** [review]

**Pull request merged**